### PR TITLE
ADD NAVBAR COMPONENT (#31)

### PR DESCRIPTION
**Pull request by abdulmaleekadams**
image

#31 The component is ready for review and the blue circle on the Figma design is display on mouse hover

### CODE REVIEW

File: src/components/header/header.jsx [hunk 1]
```diff
@@ -1,9 +1,98 @@
...
+          <a href="/">
+            <svg
+              width="29"
+              height="32"
+              viewBox="0 0 29 32"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M5.24739 0C5.43789 1.34923 6.05445 2.59947 7.00463 3.56326C7.9548 4.52704 9.18739 5.15244 10.5176 5.34567C9.18701 5.53986 7.95432 6.16617 7.0042 7.13073C6.05408 8.0953 5.43769 9.34619 5.24739 10.696C5.05831 9.34953 4.44523 8.10115 3.49959 7.13697C2.55396 6.17279 1.32642 5.54446 0 5.34567C1.33018 5.15244 2.56276 4.52704 3.51294 3.56326C4.46311 2.59947 5.07967 1.34923 5.27018 0H5.24739Z"
```
Comment (mrxshinji) [inline src/components/header/header.jsx:20]: move svg into separate file, can be inside current folder or inside assets folder

File: src/components/header/header.jsx [hunk 2]
```diff
@@ -1,9 +1,98 @@
...
...5hPN7C6cxoGETmHDp7nI16SwP7a6kTaznC+IEgEaIGJIBrgBKwYhFYCJ5xq51OeIZbIakEKgh1ndxHfu3nFNvP9Ye657QswjizHRB4znyF4k2THg/WmLrJmXpHaJDooB4rEzEVRvPLKKxeuXI4SheAf/vkfr127VlXVb37zmztvvfnud9/57X/8v5uqIaJ+bnzpc7QmWp22VccxpqRE5LCO6e7jx8+OT9974+3bV64f7Ow4a6HruqZGskQmywoFKFRBmMYcmrZb1V1oh8OSrLHWJpW2CUnYFXk5Gb351h1yRESTyWQ0KL78p0/++Z9+/2QxO+7ivQcPUfDtt99+9eJVm5KEtDiZaUgWzBppgWSMC0maRZ35lFSDsS3Qsm2Ou/S4Csdd+uzwBPJhXgwz5zPBUsyeuBtZxk+f/f7T/3J1MpoMs1lYXX3jlW//5IP80gGURWRZLRbtaoVZMShzS2TRFuNh03WsqAZ3Lh5cv/XKo6/uLY5O7XnzZn+dw8a891tZhnUlUUARMW3YLrf8lj99CUqWZTdv3vz44x8/Oz0sS193TTHI/1yMgQDK0jWhW9WmU0gMrNagcy60XZP7rPCKashdv3791mu3j56dMmAXw9lZWzhTGJdbY8vMOee9t9aqxJhSP5rJeEOats2IboUrLxzwjRSpqrW2N0Sw6YHtN+O5nGx/ELHnKoMtwFj/n30j4Lq6dk46S0CKljT1WCqBpNAkXopCVqrNOGGMKWHK8nycF/uDwYXx6Ozz31+/ei2uKpYIVguf1doOh0MSWzep6moxuHtxb8/ni/wsRWAWYwwz121tvdvZ2Rnv7R6dHp0tZm+99dYnn//xb/7ub4fD4V98/BPnXNe0wGK9J2syAlNkOSb2pj0TYEucmFljqLquaZoV0H+tOow8yLLxYNj30uZ57r3Pip5H0Lg+uRRiqJrQNnXXjkaD3f29oihY0WX+4PKl4upl8BaaOkk8Ojr6z7/53WI2q9vm2dPjT54+Nc5/9P333n79zfmjw2ZV7+clAXqXc0yxjQzBu8Jk3qGm0NZ1Aw6SpUbkpO6+ni3uz9tnXcwvX+uEkprSF2PMSpUBE1YNmGaceS8ibXv9+uUffPQDd/t2tZrnznWxqet6OZ9jwc6ScWSMWaxWQmDIibDz2f6Fg93d3VA1/x/2siAq1sHVdQAAAABJRU5ErkJggg=="
```
Comment (mrxshinji) [inline src/components/header/header.jsx:88]: would be glad if you can move this into single file. this is too long

**Review by mrxshinji — CHANGES_REQUESTED** [review]
Just some changes, needed, Excellent PR @abdulmaleekadams

File: package.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.7.1",
```
Comment (aftabrehan) [inline package.json:17]: Why are we using a library for icons?

File: src/components/header/header.css [hunk 4]
```diff
@@ -1,3 +1,150 @@
...
+  flex: 1;
+}
+
+.navbarBrand {
+  margin-right: 40px;
+}
+.gap-20 {
+  gap: 20px;
+}
+.navbarItem {
```
Comment (aftabrehan) [inline src/components/header/header.css:32]:
```suggestion
}

.gap-20 {
  gap: 20px;
}

.navbarItem {
```

File: src/index.css [hunk 5]
```diff
@@ -10,3 +10,11 @@ body {
...
   -moz-osx-font-smoothing: grayscale;
 }
+input{
+  outline: none;
+  border: none;
+}
+a{
+  text-decoration: none;
+  color: inherit;
+}
```
Comment (aftabrehan) [inline src/index.css:20]:
```suggestion
}

input{
  outline: none;
  border: none;
}

a{
  text-decoration: none;
  color: inherit;
}
```

**Review by aftabrehan — CHANGES_REQUESTED** [review]
Excellent PR, @abdulmaleekadams 💪 🚀 💯 

It just needs minor fixes and then it's good to go 🚀

File: src/components/header/header.css [hunk 6]
```diff
@@ -1,3 +1,150 @@
...
+  padding: 5px;
+  position: relative;
+  background-color: transparent;
+}
+
+.navbarSearchForm input::placeholder {
+  font-size: 13px;
+  color: #d1d1d6;
+  position: absolute;
+  letter-spacing: -0.005em;
```
[the remark this answers is not in the document]
Comment (abdulmaleekadams) [reply src/components/header/header.css:91]: I was trying out somethings before changing the style I used left only the font styling and color to overide the default styling of the placeholder

**Review by mrxshinji — APPROVED** [review]
Good work 👍

**Review by aftabrehan — APPROVED** [review]
@abdulmaleekadams LGTM 👍

**Comment by mrxshinji** [comment]
Fixes #31

**Pull request merged**